source/features/mark-merge-commits-in-list.tsx

```diff
@@ -27,7 +27,7 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 
 	const mergeCommits = [];
 	for (const [key, commit] of objectEntries(repository)) {
-		if (commit.parents.totalCount >= 2) {
+		if (commit?.parents.totalCount >= 2) {
```

Comment:
How's this possible/necessary all of the sudden?

Reply:
<img width="589" alt="image" src="https://github.com/user-attachments/assets/aa949928-80cb-4f60-aa1d-02c99420232b">

<img width="1413" alt="image" src="https://github.com/user-attachments/assets/17215d4d-55c5-429f-b047-49e8632ba050">

Reply:
That looks like the selector isn't good enough and it's causing the extension to request invalid data to the API. It should be fixed instead of altering this line

Reply:
I think reason is here, but what caused the problem.
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/bb519484-dcc4-4de3-af11-8386fe692ba1">

<img width="1353" alt="image" src="https://github.com/user-attachments/assets/7b0c7fe4-1e7d-4e58-8535-d63888a6fd28">

Reply:
I think it's getCommitLink. It should use a different link, not the title. There's another link to the commit on the side (which is where the commit hash is, that's what we want)